tests/test_deferredCharge.py

```diff
@@ -37,153 +38,231 @@ class CpCtiSolveTaskTestCase(lsst.utils.tests.TestCase):
     def setUp(self):
         self.camera = IsrMock().getCamera()
 
```

Comment:
Where did these data come from?

Reply:
I think the originals were just empirical values I had during testing, and suspect the same of the updated values.